mods/lord/Blocks/lord_lamps/src/lamps/skull_candle.lua

```diff
@@ -0,0 +1,44 @@
+local S = minetest.get_mod_translator()
+
+local function register_skull_candle()
+	local skull_candle_description = S('Skull Candle')
+
+	minetest.register_node('lord_lamps:skull_candle', {
+		description = skull_candle_description,
+		drawtype = 'mesh',
```

Comment:
выровнять бы:
```suggestion
		description = skull_candle_description,
		drawtype    = 'mesh',
```
и ниже